pydeconz/sensor.py

```diff
@@ -166,27 +167,52 @@ def __init__(self, device_id, device):
         self._status = device['state'].get('status')
         super().__init__(device_id, device)
         self._sensor_class = 'daylight'
+        self._sensor_icon = 'mdi:white-balance-sunny'
 
     @property
     def state(self):
         """Main state of sensor."""
-        return self.is_tripped
+        return self.status
```

Comment:
Missing a _ for self._status

Reply:
I don't actually want to return _status though, but instead return the "status" property that translates the _status integer to a useful string.

Reply:
Sorry. To quick of a glance. Your right ofc